doc/content/3.documentation/2.transports/4.amazon-sqs.md

```diff
@@ -5,6 +5,10 @@ title: Amazon SQS Transport
 
 # Amazon SQS
 
+::alert{type="info"}
```

Comment:
What is polymorphic dispatch? Why would I need to know this at the top of the page?

I mean, yes, I realize this is a placeholder :)